src/main/java/com/github/manolo8/darkbot/core/objects/facades/AssemblyMediator.java

```diff
@@ -121,13 +125,15 @@ public static class ItemFilter extends Reporting implements AssemblyAPI.Filter {
         private String filterName = "";
         private boolean isChecked;
         private int row, col;
+        private double x, y;
```

Comment:
it doesn't seem like anything in darkbot is using these, are you using them from the plugin? If so, please stop doing so and add methods in darkbot for the purpose.

This class should not be public, AssemblyAPI.Filter is the public interface and should be the only visible face of it.